### PR TITLE
Added support for BodyTemplate7 Display Directive

### PR DESCRIPTION
**Pull request by fuzzysb**
the template is very similar to BodyTemplate6 however can only display the title

### CODE REVIEW

File: Alexa.NET/Response/Directive/Templates/Types/BodyTemplate7.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿using Newtonsoft.Json;
+
+namespace Alexa.NET.Response.Directive.Templates.Types
+{
+    public class BodyTemplate7:IBodyTemplate
+    {
+        public string Type => "BodyTemplate6";
```
[the remark this answers is not in the document]
Comment (fuzzysb) [reply Alexa.NET/Response/Directive/Templates/Types/BodyTemplate7.cs]: It certainly should be yes. I did do this correctly the first time but i had to redo as I cloned your repo by mistake and not my fork and missed it second time around :pensive:

**Comment by fuzzysb** [comment]
I know that i am being impatient, but could we get this in place sooner rather than later. its just that i would  like to use the library as is in my new project and update via nuget rather than deviating to my forked library just for this small addition.

**Comment by stoiveyp** [comment]
@fuzzysb if you use the main nuget package, you can keep a local copy of BodyTemplate7 in the project you're using it in. Any class implementing IBodyTemplate is acceptable in the directive, so one in the project itself would work fine and save you using a forked library.

Once the PR is picked up, you can swap it out for the library version in your own time

**Comment by stoiveyp** [comment]
The following zip file contains a patch - doesn't change the code of the PR, but does add tests for body template 6 & 7, as well as ListTemplate2 for completeness. Hadn't realised they were missing until I was looking over this PR 😄 
[templatetests.zip](https://github.com/timheuer/alexa-skills-dotnet/files/1596114/templatetests.zip)

**Comment by fuzzysb** [comment]
ok i have applied the patch, thanks

**Pull request merged**